docs/UserGuide.md

```diff
@@ -64,15 +64,6 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
```

Comment:
User Guide looks good and same as what we have in google docs! 👍

Reply:
Agree!